src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -287,11 +287,15 @@ const buildWhereClause = ({ schema, query, index }): WhereClause => {
               inPatterns.push(`${listElem}`);
             }
           });
-          patterns.push(`(${name})::jsonb @> '[${inPatterns.join()}]'::jsonb`);
+          patterns.push(
+            `(${name})::jsonb @> '[${inPatterns.join(',')}]'::jsonb`
```

Comment:
the default parameter for `.join()` is a comma so this change is unnecessary

Reply:
Ah, forgot that. I'll remove.